stock_putaway_method/tests/test_putaway_method.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestPutawayMethod(SavepointCase):
```

Comment:
No real reason to use SavepointCase when no actual data is created or updated in setUpClass. Can be TransactionCase